.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+- '3.4'
```

Comment:
You may want to add 3.5, and 3.6 as well.

Reply:
So if adding 3.5 and 3.6, does it mean the tests will be run for 3 times seperately?

Reply:
Yes, three different versions are used.